app/conf/search_query_builder.conf

```diff
@@ -0,0 +1,89 @@
+#
+# Search Query Builder configuration
+#
+
+# Whether or not to display the query builder button, and include the relevant JavaScript on the search form page.
+query_builder_enabled = 0
+
+# Global config for the query builder.  See http://querybuilder.js.org/#usage for available settings.
+query_builder_global_options = {}
+
+query_builder_operators = {
+    select = [ equal, not_equal, is_empty, is_not_empty ],
+    string = [ equal, not_equal, in, not_in, begins_with, not_begins_with, contains, not_contains, ends_with, not_ends_with, is_empty, is_not_empty ],
+    integer = [ equal, not_equal, between, not_between, is_empty, is_not_empty ],
+    double = [ equal, not_equal, between, not_between, is_empty, is_not_empty ],
+    date = [ equal, not_equal, between, not_between, is_empty, is_not_empty ],
+    time = [ equal, not_equal, between, not_between, is_empty, is_not_empty ],
+    datetime = [ equal, not_equal, between, not_between, is_empty, is_not_empty ]
+}
+
+# Whether or not to display the query builder per table.  Both this and `query_builder_enabled` must be `1` to display
+# the query builder for a given table.
+query_builder_enabled_ca_objects = 1
+query_builder_enabled_ca_object_lots = 1
+query_builder_enabled_ca_entities = 1
+query_builder_enabled_ca_places = 1
+query_builder_enabled_ca_collections = 1
+query_builder_enabled_ca_storage_locations = 1
+query_builder_enabled_ca_loans = 1
+
+# Per-table list of field/filter names to move to the top of the query builder's list, in the order given.
+query_builder_priority_ca_objects = [
+    ca_objects.idno,
+    ca_object_labels.name,
+    ca_objects.type_id,
+    ca_objects.status,
+    _fulltext
+]
+
+query_builder_priority_ca_object_lots = [
+    ca_object_lots.idno_stub,
+    ca_object_lots.type_id,
+    ca_object_lots.status,
+    _fulltext
+]
+
+query_builder_priority_ca_entities = [
+    ca_entities.idno,
+    ca_entities.type_id,
+    ca_entities.status,
+    _fulltext
+]
+
+query_builder_priority_ca_places = [
+    ca_places.idno,
+    ca_places.type_id,
+    ca_places.status,
+    _fulltext
+]
+
+query_builder_priority_ca_collections = [
+    ca_collections.idno,
+    ca_collections.type_id,
+    ca_collections.status,
+    _fulltext
+]
+
+query_builder_priority_ca_storage_locations = [
+    ca_storage_locations.idno,
+    ca_storage_locations.type_id,
+    ca_storage_locations.status,
+    _fulltext
+]
+
+query_builder_priority_ca_loans = [
+    ca_loans.idno,
+    ca_loans.type_id,
+    ca_loans.status,
+    _fulltext
+]
+
+# Per-table list of field/filter names to exclude from the query builder.
```

Comment:
I think I should just take "field" out here and call them filters, that's what the query builder calls them.  And maybe put a comment somewhere that intrinsic fields, attributes and access points are all mapped to filters.